.github/scripts/validate-deploy.sh

```diff
@@ -50,21 +50,25 @@ else
   sleep 30
 fi
 
-DEPLOYMENT="${COMPONENT_NAME}-${BRANCH}"
+RESOURCE01="db2oltp"
+
 count=0
-until kubectl get deployment "${DEPLOYMENT}" -n "${NAMESPACE}" || [[ $count -eq 20 ]]; do
-  echo "Waiting for deployment/${DEPLOYMENT} in ${NAMESPACE}"
+
+until kubectl get pods -l icpdsupport/addOnId=${RESOURCE01} -n "${NAMESPACE}" 1> /dev/null 2> /dev/null || [[ $count -eq 20 ]]; do
+  echo "Waiting for pod ${RESOURCE01} in ${NAMESPACE}"
   count=$((count + 1))
-  sleep 15
+  sleep 45
 done
 
+
+
 if [[ $count -eq 20 ]]; then
-  echo "Timed out waiting for deployment/${DEPLOYMENT} in ${NAMESPACE}"
-  kubectl get all -n "${NAMESPACE}"
-  exit 1
+  echo "Timed out waiting for job ${RESOURCE} in ${NAMESPACE}"
+  kubectl get all -n "${NAMESPACE}" 
+  
 fi
 
-kubectl rollout status "deployment/${DEPLOYMENT}" -n "${NAMESPACE}" || exit 1
+kubectl get all -l icpdsupport/addOnId=${RESOURCE01} -n "${NAMESPACE}"|| exit 1
```

Comment:
`kubectl get all -l` finds all the resources in the namespace with the label but does not check that they actually exist nor does it check that they are healthy. As mentioned above, if possible you should look for the controller that is creating the pods (StatefuleSet or Deployment) and do `kubectl rollout status` to wait for it to provision all of its pods and for those pods to be healthy. If that is not possible then this logic needs to look for all the pods with the label and check their status.

Reply:
Got it. Doing the kubectl rollout status  of the statefulset created by the job